console.js

```diff
@@ -91,6 +91,8 @@ exports.commands = {
 		},
 		forcestart: 'start',
 		start: function (target, room, user, connection, cmd, message) {
+			if (room.battle) return this.errorReply("Consoles are not allowed in Battle Rooms.");
+			if (user.console) return this.errorReply("You already have console active with id " + user.consoleId);
```

Comment:
The reason we allow forcestart is for switching games in an active console (ex `/forcestart sggamealpha`)